common/constant/key.go

```diff
@@ -50,7 +50,8 @@ const (
 	PROTOCOL_KEY             = "protocol"
 	PATH_SEPARATOR           = "/"
 	//DUBBO_KEY                = "dubbo"
-	SSL_ENABLED_KEY = "ssl-enabled"
+	SSL_ENABLED_KEY  = "ssl-enabled"
+	ParameterTypeKey = "parameter-type-names" // ParameterType key used in dapr, to tranfer rsp param type
```

Comment:
Maybe follow the code`SSL_ENABLED_KEY `.

Reply:
fixed